sarathi/model_executor/attention/base_attention_wrapper.py

```diff
@@ -37,6 +40,14 @@ def get_timer(self, operation: OperationMetrics, layer_id: Optional[int] = None)
         if self._timers.get((operation, layer_id)) is None:
             self._timers[(operation, layer_id)] = CudaTimer(operation, layer_id)
         return self._timers.get((operation, layer_id))
+    
+    @abstractmethod 
+    def init_gpu_cache(self, num_gpu_blocks: int) -> None:
+        pass
+    
+    @abstractmethod
+    def get_cache_block_size(self) -> int:
```

Comment:
is this required anymore?

Reply:
let's just keep this method and keep we can have generic logic in init gpu cache

Reply:
Moved the init_gpu_cache logic into this base class